### ISSUE TITLE
List shared files in a room

### ISSUE THREAD

**Issue by Vetgans** [element-hq/element-x-ios#3332]
Same as https://github.com/element-hq/element-x-android/issues/3494

**Comment by johansmitsnl** [comment]
Also made a meta issue here: https://github.com/element-hq/element-meta/issues/2607

**Comment by b1ggi** [comment]
this is already implemented?

**Comment by stefanceriu** [comment]
Yes, since https://github.com/element-hq/element-x-ios/pull/3588
